src/RestrictedPython/Guards.py

```diff
@@ -266,3 +266,15 @@ def guarded_setattr(object, name, value):
 def guarded_delattr(object, name):
     delattr(full_write_guard(object), name)
 safe_builtins['delattr'] = guarded_delattr
+
+
+def guarded_unpack_sequence(it, spec, _getiter_):
+    ret = list(_getiter_(it))
```

Comment:
This function should have a docstring. From reading the code I do not understand how it is to be used.